stock_orderpoint_automatic_creation/models/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Daniel Campos <danielcampos@avanzosc.es> - Avanzosc S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import product
```

Comment:
The rule is the model name, and the model name is product.product, not only product